src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -78,6 +80,7 @@ fun managedIndexConfigIndexRequest(index: String, uuid: String, policyID: String
     return IndexRequest(INDEX_MANAGEMENT_INDEX)
             .id(uuid)
             .create(true)
+            .routing(managedIndexConfig.indexUuid)
```

Comment:
Is this just for being explicit? As by default elasticsearch uses the document id for routing and the document id is being set to the index uuid in this case.

Reply:
don't know that. but should be good to be explicit here